tests/integration/test_discourse.py

```diff
@@ -17,6 +17,9 @@
 pytestmark = pytest.mark.discourse
 
 
+SLUG_REGEX = r"\/t\/[\w-]*\/"
```

Comment:
why not `r"\/t\/[\w-]+\/"` ?

Reply:
it shouldn't matter in this case although it is probably slightly better. Changed it to using `+`